api/lib/domain/models/Examiner.js

```diff
@@ -30,6 +36,25 @@ class Examiner {
       correctedAnswer.result = AnswerStatus.TIMEDOUT;
     }
 
+    if (Boolean(answer.isFocusedOut) !== hasAssessmentRecordedFocusedout) {
+      logger.warn(
+        {
+          subject: 'focusOut',
+          challengeFormat,
+          challengeId: answer.challengeId,
+          assessmentId: answer.assessmentId,
+        },
+        `Received answer.isFocusedOut=${answer.isFocusedOut} even though ${
+          hasAssessmentRecordedFocusedout ? 'a ' : 'no '
+        }focusedout event has been recored`
+      );
```

Comment:
Dans les logs tu peux faire de la substitution un peu plus facilement :
```suggestion
     logger.warn(
        {
          subject: 'focusOut',
          challengeFormat,
          challengeId: answer.challengeId,
          assessmentId: answer.assessmentId,
        },
        `Received answer.isFocusedOut=${answer.isFocusedOut} even though %s focusedout event has been recorded`, hasAssessmentRecordedFocusedout ? 'a ' : 'no '
        }
      );
```